c31-pryhaven/nutshell2
# My event form

# Description

The form for adding new events has been completed and is functioning as intended.  Upon clicking the Submit button the user is redirected back to the events page and the DOM is refreshed with the new event.

Applies to issue # (issue number)
3.1 & 3.2

## Type of change

Please delete options that are not relevant.

- [ x ] Bug fix (non-breaking change which fixes an issue)

# Testing Instructions

run `git fetch --all` in terminal
pull down my-event-form from github.
take data from the example database and plug it into nutshell.json.
run the command `json-server -p 8088 -w nutshell.json` at ./nutshell2/api
go to root path and run `npm start`
go to browser and click on `events` on the nav bar.
click on the add event button and fill out form.
hit the submit button and you should be redirected to the events page with your newly added event.


# Checklist:

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ x ] My changes generate no new warnings or errors
- [ x ] I have added test instructions that prove my fix is effective or that my feature works
